### PR TITLE
Hp prod type access

### PR DESCRIPTION
**Pull request by hunterphillips**
**Changes**:
Complete Product Type routes and GET, POST, PUT, DELETE functions

**Testing**:
nodemon app.js
Use Postman to POST, PUT, & DELETE entries

**Postman instructions**:
URL: http://localhost:8080/api/product-types
Header
'Key' : Content-Type,
'Value' : application/json

Body
Select 'Raw'
Insert following objects -
**DELETE**
{
"id": 2
}

**POST**
{
"name": "kinky stuff"
}

**PUT**
URL: http://localhost:8080/api/product-types/1
{
"name": "graves"
}
**Expected results**:
User should be able to DELETE, POST, GET, PUT product_type entries

**Tickets**:
Completes ticket #5

### CODE REVIEW

**Review by dhazlettjr — APPROVED** [review]
GET - working
POST - working
DELETE - working
PUT - working

File: models/Product-types.js
Comment (e-x-wilson) [inline models/Product-types.js]: Hello Hunter. Hope your Friday is going well. I know you are on lunch, but I would like to request that you remove this console.log as soon as possible so we can move forward with merging this PR. See you back at the office

- Eli

**Review by electriceeli — CHANGES_REQUESTED** [review]

**Comment by electriceeli** [comment]
GET all prod types - 😋 
GET one prod type - ☯️ 
DELETE prod type - 🛰 
POST prod type -  **_k i n k y_**
PUT prod type attribute - 💀 

*Will approve once requested changes are submitted

**Review by electriceeli — APPROVED** [review]
Excellente, senõr

**Pull request merged**
